exec/Chainweb/Env.hs

```diff
@@ -158,11 +159,18 @@ readNodeDbPath = eitherReader $ \case
 data Command
     = Server ServerEnv
     | Listen
-    | Backfill (Maybe Int)
+    | Backfill BackfillArgs
     | Gaps (Maybe Int)
     | Single ChainId BlockHeight
     deriving (Show)
 
+data BackfillArgs = BackfillArgs
+  { _backfillArgs_delayMicros :: Maybe Int
+  , _backfillArgs_onlyEvents :: Bool
+  , _backfillArgs_eventChunkSize :: Maybe Integer
+  , _backfillArgs_chainwebVersion :: ChainwebVersion
```

Comment:
We should remove this field.  This information already exists in the `NodeInfo` data structure from chainweb-api (https://github.com/kadena-io/chainweb-api/blob/master/lib/Chainweb/Api/NodeInfo.hs#L19).